src/pages/en/reference/api-reference.md

```diff
@@ -701,6 +701,147 @@ Pagination will pass a `page` prop to every rendered page that represents a sing
 | `page.url.prev`    | `string \| undefined` | Get the URL of the previous page (will be `undefined` if on page 1).                                                              |
 | `page.url.next`    | `string \| undefined` | Get the URL of the next page (will be `undefined` if no more pages).                                                              |
 
+## Content Collections (Experimental)
+
+<p>
+  <Since v="1.7.0" />
+</p>
+
+Content collections offer APIs to configure and query your Markdown or MDX documents in `src/content/`. For feature rundowns and usage examples, [see our content collections guide](/en/guides/content-collections/).
```

Comment:
```suggestion
Content collections offer APIs to configure and query your Markdown or MDX documents in `src/content/`. For features and usage examples, [see our content collections guide](/en/guides/content-collections/).
```
nit: Now that the example pulled my attention back up here, I think we can go a little less colloquial on the API reference page.

But, I think.... after that we can update branch?? 🎉